src/FSharp.Markdown/app.config

```diff
@@ -1,11 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is there still a need for app config here?

Reply:
It remained in the repo after I removed it from project. I rm it definitely